x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/__mocks__/request_responses.ts

```diff
@@ -224,69 +245,91 @@ export const getFindResultWithMultiHits = ({
   };
 };
 
-export const getImportRulesRequest = (payload?: Buffer): ServerInjectOptions => ({
-  method: 'POST',
-  url: `${DETECTION_ENGINE_RULES_URL}/_import`,
-  headers: {
-    'Content-Type': `multipart/form-data; boundary=${TEST_BOUNDARY}`,
-  },
-  payload,
-});
-
-export const getImportRulesRequestOverwriteTrue = (payload?: Buffer): ServerInjectOptions => ({
-  method: 'POST',
-  url: `${DETECTION_ENGINE_RULES_URL}/_import?overwrite=true`,
-  headers: {
-    'Content-Type': `multipart/form-data; boundary=${TEST_BOUNDARY}`,
-  },
-  payload,
-});
-
-export const getDeleteRequest = (): ServerInjectOptions => ({
-  method: 'DELETE',
-  url: `${DETECTION_ENGINE_RULES_URL}?rule_id=rule-1`,
-});
-
-export const getDeleteRequestById = (): ServerInjectOptions => ({
-  method: 'DELETE',
-  url: `${DETECTION_ENGINE_RULES_URL}?id=04128c15-0d1b-4716-a4c5-46997ac7f3bd`,
-});
-
-export const getCreateRequest = (): ServerInjectOptions => ({
-  method: 'POST',
-  url: DETECTION_ENGINE_RULES_URL,
-  payload: {
-    ...typicalPayload(),
-  },
-});
-
-export const getSetSignalStatusByIdsRequest = (): ServerInjectOptions => ({
-  method: 'POST',
-  url: DETECTION_ENGINE_SIGNALS_STATUS_URL,
-  payload: {
-    ...typicalSetStatusSignalByIdsPayload(),
-  },
-});
-
-export const getSetSignalStatusByQueryRequest = (): ServerInjectOptions => ({
-  method: 'POST',
-  url: DETECTION_ENGINE_SIGNALS_STATUS_URL,
-  payload: {
-    ...typicalSetStatusSignalByQueryPayload(),
-  },
-});
-
-export const getSignalsQueryRequest = (): ServerInjectOptions => ({
-  method: 'POST',
-  url: DETECTION_ENGINE_QUERY_SIGNALS_URL,
-  payload: { ...typicalSignalsQuery() },
-});
-
-export const getSignalsAggsQueryRequest = (): ServerInjectOptions => ({
-  method: 'POST',
-  url: DETECTION_ENGINE_QUERY_SIGNALS_URL,
-  payload: { ...typicalSignalsQueryAggs() },
-});
+export const ruleStatusRequest = () =>
+  requestMock.create({
+    method: 'get',
+    path: `${DETECTION_ENGINE_RULES_URL}/_find_statuses`,
+    query: { ids: ['someId'] },
+  });
+
+export const getImportRulesRequest = (hapiStream?: HapiReadableStream) =>
+  requestMock.create({
+    method: 'post',
+    path: `${DETECTION_ENGINE_RULES_URL}/_import`,
+    headers: {
+      'Content-Type': 'multipart/form-data; boundary=???',
+    },
+    body: { file: hapiStream },
+  });
+
+export const getImportRulesRequestOverwriteTrue = (hapiStream?: HapiReadableStream) =>
+  requestMock.create({
+    method: 'post',
+    path: `${DETECTION_ENGINE_RULES_URL}/_import`,
+    headers: {
+      'Content-Type': 'multipart/form-data; boundary=???',
+    },
```

Comment:
I removed both of these headers that have `boundary=???` and nothing breaks, so I would just remove this one and the one below it. Better to not have additional stuff in tests that aren't needed.

Reply:
Makes sense, since we're no longer testing that Hapi converts our requests to streams?

Reply:
https://github.com/elastic/kibana/pull/58292/commits/4c540919a3d90ae1a74512bada1c87707cdfe7ae